movieapp/src/containers/Header/Header.tsx

```diff
@@ -39,7 +40,8 @@ export function Header(props: HeaderProps): JSX.Element {
         {t(CONSTANTS.ROULETTE)}
       </h1>
       <Button type={ButtonTypes.button} onClick={openModal} title={t(CONSTANTS.ADD_MOVIE)} />
-      <SearchForm />
+      {!props.expandedMovie ? <SearchForm /> : ''}
```

Comment:
for if render we use && operator in react
```
{!props.expandedMovie && <SearchForm />}
```

Reply:
fixed